test-suite/shared/src/test/scala/org/scalajs/testsuite/niobuffer/BufferFactory.scala

```diff
@@ -55,8 +59,7 @@ object BufferFactory {
 
     implicit def elemFromInt(value: Int): ElementType = value.toByte
 
-    implicit def elemToJSAny(elem: ElementType): js.Any =
-      js.Any.fromByte(elem)
+    implicit def elemToAnyRef(elem: ElementType): AnyRef = Byte.box(elem)
```

Comment:
It does not allow the implicit conversion in this case. It says:
```scala
[error]  found   : ByteBufferFactory.this.ElementType
[error]     (which expands to)  Byte
[error]  required: AnyRef
[error] Note: an implicit exists from scala.Byte => java.lang.Byte, but
[error] methods inherited from Object are rendered ambiguous.  This is to avoid
[error] a blanket implicit which would convert any scala.Byte to any AnyRef.
[error] You may wish to use a type ascription: `x: java.lang.Byte`.
[error]     implicit def elemToAnyRef(elem: ElementType): AnyRef = elem
[error]                                                            ^
```
I used the `box` because it was more explicit. I could use the ascription if you prefer.